src/main/resources/application.yml

```diff
@@ -0,0 +1,23 @@
+spring:
```

Comment:
Please check with Rafi, if he already has an application.yml or .properties, then please use that.

Reply:
We can finalize what data-serialization language for our application configuration. I think YML is much readable and being used across our application such as Kubernetes